### PR TITLE
New templates: alcatel_sros_ping.textfsm and alcatel_sros_show_service_id_interface.textfsm. Modified alcatel_sros_show_service_sap-using.textfsm

### PR DESCRIPTION
**Pull request by jorlandobr**
Added template ands test for Alcatel (Nokia) SROS for the  `ping`, `ping rapid` and `show service id interface` commands. Index updated.

Modified `show service sap using` changing the value **INGRESS_QOS** from digit to string to avoid errors when trucated rows present the `*` character.

### CODE REVIEW

**Comment by jorlandobr** [comment]
> Added template ands test for Alcatel (Nokia) SROS for the **"ping"**, **"ping rapid"** and **"show service id interface"** commands. Index updated.
> 
> Modified **"show service sap using"** changing the value **INGRESS_QOS** from digit to string to avoid errors when trucated rows present the ***** character.

Modified the ping template, so a single template works with "ping" and "ping rapid" commands.

File: ntc_templates/templates/alcatel_sros_show_service_id_interface.textfsm [hunk 1]
```diff
@@ -0,0 +1,24 @@
+Value IFACE_NAME (\S+)
```
Comment (jvanderaa) [inline ntc_templates/templates/alcatel_sros_show_service_id_interface.textfsm:1]:
```suggestion
Value INTERFACE_NAME (\S+)
```

File: ntc_templates/templates/alcatel_sros_show_service_id_interface.textfsm [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+Value List IP_ADDRESS (\S+)
+Value List PFXSTATE (\S+)
+Value OPR_STATE_V4V6 (\S+)
+Value TYPE (\S+)
+Value PORT_SAP_ID (\S+)
+Value ADM_STATE (\S+)
+Value IFACE_QTY (\d+)
```
Comment (jvanderaa) [inline ntc_templates/templates/alcatel_sros_show_service_id_interface.textfsm:8]:
```suggestion
Value INTERFACE_QTY (\d+)
```

**Review by jvanderaa — CHANGES_REQUESTED** [review]
Please update to full names.

File: ntc_templates/templates/alcatel_sros_show_service_id_interface.textfsm (hunk 1 above)
Comment (jorlandobr) [reply ntc_templates/templates/alcatel_sros_show_service_id_interface.textfsm]: Changes applied.

File: ntc_templates/templates/alcatel_sros_show_service_id_interface.textfsm (hunk 2 above)
Comment (jorlandobr) [reply ntc_templates/templates/alcatel_sros_show_service_id_interface.textfsm]: Changes applied.

**Review by jorlandobr — COMMENTED** [review]
Changes made as requested

File: ntc_templates/templates/alcatel_sros_show_service_id_interface.textfsm [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+  ^\S+
+  ^. -> Error
+
+Interface
+  ^\s+IP-Address\s+PfxState
+  ^-+
+  ^.*?\s+(Up|Down) -> Continue.Record
+  ^${INTERFACE_NAME}\s+${ADMIN_STATE}\s+${OPER_STATE_V4}\/${OPER_STATE_V6}\s+${TYPE}\s+${PORT_SAP_ID} 
+  ^\s+${IP_ADDRESS}\s+${PFX_STATE}
+  ^Interfaces -> Done
```
Comment (jmcgill298) [inline ntc_templates/templates/alcatel_sros_show_service_id_interface.textfsm:25]:
```suggestion
  ^Interfaces\s*:\s+\d+\s*$$
```

We are at the end of the data, so I don't see a need to move to another state to move to the implicit `End -> Record` state.

File: ntc_templates/templates/alcatel_sros_show_service_sap-using.textfsm [hunk 4]
```diff
@@ -22,6 +22,7 @@ SAPS
   ^=+
   ^Number\s+of
   ^\s*$$
+  ^\S+
```
Comment (jmcgill298) [inline ntc_templates/templates/alcatel_sros_show_service_sap-using.textfsm:25]: What is the reason for this change? Can you update the test case or add a test case with data the exemplifies the issue this is fixing?

**Review by jmcgill298 — CHANGES_REQUESTED** [review]
Thanks for the PR, just a couple of small changes requested

File: ntc_templates/templates/alcatel_sros_show_service_sap-using.textfsm [hunk 5]
```diff
@@ -22,6 +22,8 @@ SAPS
   ^=+
   ^Number\s+of
   ^\s*$$
+  # added to avoid errors with trucated values alert message
```
Comment (jmcgill298) [inline ntc_templates/templates/alcatel_sros_show_service_sap-using.textfsm:25]: Is this referring to `* indicates that the corresponding row element may have been truncated.`?

**Comment by jmcgill298** [comment]
Did not hear back on questions, so replaced with https://github.com/networktocode/ntc-templates/pull/1729

**Pull request closed without merging**